ecs.php

```diff
@@ -9,25 +9,28 @@
 use Symplify\EasyCodingStandard\Config\ECSConfig;
 use Symplify\EasyCodingStandard\ValueObject\Option;
 
-return static function (ECSConfig $containerConfigurator): void {
-    $containerConfigurator->import('vendor/sylius-labs/coding-standard/ecs.php');
+return static function (ECSConfig $ecsConfig): void {
+    $ecsConfig->paths([
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+    ]);
+
+    $ecsConfig->import('vendor/sylius-labs/coding-standard/ecs.php');
 
-    $containerConfigurator
-        ->services()
-        ->set(HeaderCommentFixer::class)->call('configure', [[
-            'location' => 'after_open',
-            'header' =>
-'This file is part of the Sylius package.
+    $ecsConfig->ruleWithConfiguration(HeaderCommentFixer::class, [
+        'location' => 'after_open',
+        'header' =>
+            'This file is part of the Sylius package.
 
 (c) Paweł Jędrzejewski
```

Comment:
@Rafikooo you should update licences in technical libs as well, shouldn't you?

Reply:
yes, Lukasz, it will be another PR.

Reply:
@lchrusciel, sure! 🚀